### PR TITLE
Update README.md to encourage using token instead of password.

### PR DESCRIPTION
**Pull request by hamelsmu**
@psychemedia what do you think?

### CODE REVIEW

**Comment by hamelsmu** [comment]
closes #56

**Review by choldgraf — COMMENTED** [review]
I'm +1 if @psychemedia thinks this would have made things clearer!

File: README.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ See the [examples](#examples) section is very helpful for understanding the inpu
 - `DOCKER_USERNAME`:
     description: Docker registry username
 - `DOCKER_PASSWORD`:
-    description: Docker registry password or [access token](https://docs.docker.com/docker-hub/access-tokens/)
+    description: Docker registry password or [access token](https://docs.docker.com/docker-hub/access-tokens/).  **If using DockerHub, we recommend using an access token instead of your password for security reasons.**
```
Comment (psychemedia) [inline README.md:58]: how about: *using an access token in this field instead of*

(Sometimes people need to be told really explicitly!)

Comment (hamelsmu) [reply README.md:58]: Do you mind making a commit suggestion?  Not sure I follow

**Comment by hamelsmu** [comment]
@psychemedia I made an edit that I think addresses your comment, however, feel free to open a new PR if I didn't do a good job 🙇

**Pull request merged**